src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -44,6 +45,11 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             "Missing scanned documents in exception record"
         );
 
+        final List<String> warnings =
+            caseUpdateRequest.caseUpdateDetails == null
+                ? emptyList()
+                : caseUpdateDetailsValidator.getWarnings(caseUpdateRequest.caseUpdateDetails);
```

Comment:
it seems weird that validation happen only sometimes. Perhaps it could take the entire request data?